Z3Scheduler/timetableZ3.py

```diff
@@ -58,9 +58,13 @@ def init_variables(self) :
             for sectional_lesson in value.sectionals :
                 self.lessons_by_day[sectional_lesson.day - 1].append(sectional_lesson)
                 self.sects.append(sectional_lesson)
+            
+            # Build hashmaps to facilitate solving
 
             self.build_hashmaps()
```

Comment:
build_hashmaps should stay outside of for loop.

Reply:
Fixed